src/main/java/Duke.java

```diff
@@ -1,10 +1,36 @@
+import java.io.FileNotFoundException;
 public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+    private Storage storage;
+    private Parser parser;
+    private UI ui;
+    private TaskList taskList;
+
+    public Duke(String filepath) throws Exception {
+        this.storage = new Storage(filepath);
+        try {
+            this.taskList = new TaskList(storage);
+        } catch (FileNotFoundException error) {
+            this.taskList = new TaskList();
+        }
+
+        this.ui = new UI(taskList);
+        this.parser = new Parser(taskList, storage, ui);
+    }
+
+    public void run() throws Exception {
+        ui.printIntro();
+        storage.loadTasks();
+        storage.loadTasks();
```

Comment:
Is there a reason to repeat the same process a second time?

Reply:
Dear Jun Hong T,

I believe this was a careless mistake, which I will rectify immediately.
Thank you Jun Hong T, very cool!

Best regards
Roby Tanama